static/app/views/settings/project/projectOwnership/addCodeOwnerModal.tsx

```diff
@@ -151,13 +158,36 @@ class AddCodeOwnerModal extends React.Component<Props, State> {
   render() {
     const {Header, Body, Footer, closeModal} = this.props;
     const {codeownerFile, error, errorJSON} = this.state;
-    const {codeMappings} = this.props;
+    const {codeMappings, integrations, organization} = this.props;
+    const baseUrl = `/settings/${organization.slug}/integrations`;
+
     return (
       <React.Fragment>
         <Header closeButton onHide={closeModal}>
           <h4>{t('Add Code Owner File')}</h4>
         </Header>
         <Body>
+          {!codeMappings.length && (
+            <React.Fragment>
+              <div>
+                {t(
+                  "Configure stack trace linking to add your CODEOWNERS file. Select the integration you'd like to use for mapping:"
```

Comment:
I don't think we should say 'stack trace linking' since that's the feature, I think we should use 'code mapping' instread.